src/Microsoft.Framework.Runtime/NuGet/NETPortable/NetPortableProfile.cs

```diff
@@ -42,27 +42,27 @@ public NetPortableProfile(string frameworkDirectory, string name, IEnumerable<Fr
         {
             if (String.IsNullOrEmpty(frameworkDirectory))
             {
-                throw new ArgumentNullException( "version");
```

Comment:
If you see dependency issues, do `dnu restore` before your `dnu build`, or simply do `./build.cmd` in root dir and it should take care of everything for you.